src/libraries/System.Drawing.Common/src/System/Drawing/PropertyItemInternal.cs

```diff
@@ -1,119 +1,19 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Runtime.InteropServices;
+
 namespace System.Drawing.Imaging
 {
-    using System.Runtime.InteropServices;
-
-    // sdkinc\imaging.h
-    [StructLayout(LayoutKind.Sequential)]
-    internal sealed class PropertyItemInternal : IDisposable
+#pragma warning disable CS0649
```

Comment:
Maybe just put it around the uninitialized field?

Reply:
Updated